UI/ui_pet.csv

```diff
@@ -122,70 +122,70 @@ Warning_MaxAll_Title#0,"""Egg Synthesis"""
 DesideYesNo_MsgAdd_Title#0,"""Egg Synthesis"""
-DesideYesNo_MsgAdd_Header#0,"""ペット「$(0)」に<br>$(1)を合成します。"""
-DesideYesNo_MsgAdd_Lock#0,"""<yellow><br>ロック状態のアイテムが選択されています。<c>"""
+DesideYesNo_MsgAdd_Header#0,"""Synthesizing $(1)<br>into pet \""$(0)\""."""
+DesideYesNo_MsgAdd_Lock#0,"""<yellow><br>You've selected an item that's locked.<c>"""
 DesideYesNo_MsgAdd_Rare#0,"""<yellow><br>レアリティ$(0)以上のアイテムが選択されています。<c>"""
-DesideYesNo_MsgAdd_MaxLv#0,"""<yellow><br>$(0)のレベルが上限のため、<br>経験値は獲得できません。<c>"""
+DesideYesNo_MsgAdd_MaxLv#0,"""<yellow><br>$(0)'s level is maxed,<br>so it will not gain experience.<c>"""
 DesideYesNo_MsgAdd_MaxParam#0,"""<yellow><br>$(0)のプラス値が上限のため、<br>上がらないプラス値があります。<c>"""
 DesideYesNo_MsgAdd_ReturnedEgg#0,"""<br>※これは、<c ff4f3f>「ペットをエッグに戻す」で作られたエッグ<c><br>\u3000です。本当に素材とするか、<yellow>再度ご確認ください。<c>"""
-DesideYesNo_MsgAdd_Footer#0,"""<yellow><c><br>エッグ合成を実行してもよろしいですか？"""
-Composition_Error_NoItem1#0,"""エッグ合成できるアイテムがありません。"""
+DesideYesNo_MsgAdd_Footer#0,"""<yellow><c><br>Are you sure you want to perform Egg Synthesis?"""
+Composition_Error_NoItem1#0,"""There are no items that can be used for Egg Synthesis."""
 Composition_After_Title#0,"""Egg Synthesis"""
 Composition_After_ReleaseCandy_Msg#0,"""Lv.50を超えたため、<br>ペットのキャンディーボックスから<br>ゼリーキューブが解放されました。"""
 Composition_After_LimitBreak_Msg#0,"""Lv.100に到達しました。<br><br>ゲートエリアにいるピエトロに話しかけて、<br>$(0)を<yellow>限界突破<c>しましょう。"""
-Composition_After_MaxLv_Msg#0,"""レベル上限に到達しました。"""
+Composition_After_MaxLv_Msg#0,"""Maximum level reached."""
 Title#4,"""Release Pet"""
 Message#0,"""$(0) will be released.<br>Continue?<br><br><c ff4f3f>※Released pets will be removed from the [Pet List].<br>This cannot be reversed.<c>"""
 Success#0,"""Your pet has been released."""
 Message2#0,"""This pet has a <yellow>7+ rarity<c>.<br>Do you really want to release it?"""
 Title#5,"""Limit Break"""
-Message#1,"""以下のペットの限界突破を行います。<br>よろしいですか？<br><br>※限界突破すると、<yellow>Lv.101以上<c>に成長できます。<br>※限界突破後は、エッグ合成で<yellow>低レアリティの<br>エッグ<c>で得られる経験値が大幅に少なくなります。"""
+Message#1,"""以下のペットの限界突破を行います。<br>Are you sure?<br><br>※限界突破すると、<yellow>Lv.101以上<c>に成長できます。<br>※限界突破後は、エッグ合成で<yellow>低レアリティの<br>エッグ<c>で得られる経験値が大幅に少なくなります。"""
 Success#1,"""対象のペットが、限界突破に成功しました。"""
-ErrorServerBug#0,"""サーバーとの通信に失敗しました。<br><br>時間をおいて再度お試しください。"""
-ErrorClientBug#0,"""予期せぬ問題が発生しました。<br><br>時間をおいて再度お試しください。"""
-ErrorDBAccess#0,"""データベースのアクセスに失敗しました。<br><br>時間をおいて再度お試しください。"""
+ErrorServerBug#0,"""Failed to communicate with the server.<br><br>Please try again later."""
+ErrorClientBug#0,"""An unexpected error occurred.<br><br>Please try again later."""
+ErrorDBAccess#0,"""Database access failed.<br><br>Please try again later."""
 ErrorPetTypeInvalid#0,"""ペットの種類が正しくありません。"""
-ErrorPetHasFound#0,"""既にペットを所持しています。"""
-ErrorPetNotFound#0,"""ペットが見つかりませんでした。"""
+ErrorPetHasFound#0,"""Pet is already owned."""
```

Comment:
Consider rewriting this, sounds really off. Something like "You already own this pet."